ci-operator/config/openshift/release/openshift-release-master__nightly-4.14.yaml

```diff
@@ -410,6 +419,14 @@ tests:
   steps:
     cluster_profile: vsphere-multizone
     workflow: openshift-e2e-vsphere-upi-zones
+- as: e2e-vsphere-cgroupsv1
```

Comment:
Ditto, vsphere jobs use e2e-vsphere-ipi, our naming hasn't always been the most consistent but please try to match what's in use along with https://docs.ci.openshift.org/docs/how-tos/naming-your-ci-jobs/

Reply:
e2e-vsphere-ipi-ovn-cgroupsv1